app/javascript/react/components/MonsterShow.js

```diff
@@ -0,0 +1,60 @@
+import { Binding } from "@babel/traverse";
+import React, { useState, useEffect } from "react";
+
+const MonsterShow = (props) => {
+
+  const [monster, setMonster] = useState({})
+
+  const getMonster = async () => {
+    try{
+      let monsterId = props.match.params.id
+      const response = await fetch(`/api/v1/monsters/${monsterId}`)
+      if (!response.ok){
+        const errorMessage = `${response.status} (${response.statusText})`
+        const error = new Error(errorMessage)
+        throw error
+      } 
+      const fetchedMonster = await response.json()
+      setMonster(fetchedMonster.monsters) 
+    }
+    catch (err){
+      console.log(`Error in fetch: ${err.message}`)
+    }
+  }
+
+  useEffect(() => {
+    getMonster()
+  },[])
+
+  let classificationDiv = `Classification: ${monster.classification}`
+  let habitatDiv = `Habitat: ${monster.habitat}`
+
+  if (!monster.classification) {
+    classificationDiv = ""
+  }
+  if (!monster.habitat) {
+    habitatDiv = ""
+  }
+
```

Comment:
I would flip the logic used on 29 - 37.  Instead of first trying to set the `classificationDiv` to the monsters classification and then re assign it to an empty string if the monster's classification does not exist, we should do the opposite.

Set it to an empty string for all cases and then only reassign it if,
```js
if (monster.classification) {
  classificationDiv = `Classification: ${monster.classification}`
}
```

Reply:
Done